test/TestInternalGrains/HashBasedPlacementGrain.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Orleans.Placement;
+
+namespace UnitTests.Grains
+{
+    [HashBasedPlacement(true)]
```

Comment:
Nevermind, it looks like passing `true` is actually not the default. In any case, is this still valid? Is it possible to have tests without any test-specific branching logic for this?